vnext/Microsoft.ReactNative/RedBoxHandler.idl

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import "ReactNativeHost.idl";
+
+namespace Microsoft.ReactNative.RedBox {
+
+  enum ErrorType {
+    JavaScriptFatal, // A JS Exception was thrown or otherwise fatal error
+    JavaScriptSoft, // An error coming from JS that isn't fatal, such as console.error
+    Native,
+  };
+
+  [webhosthidden] interface IErrorFrameInfo {
```

Comment:
we could use the native WinRT types for exception/error management that are already understood by debuggers: ILanguageExceptionStackBackTrace and IRestrictedErrorInfo

Reply:
I dont think these types align with the Infomation that react-native is providing.  RN is already doing some of the logic that debuggers would traditionally do, such as communicating with the bundle server and doing sourcemap lookups to determine the real stack info.  The frame information is empty for native errors.